draft-ietf-quic-transport.md

```diff
@@ -1688,8 +1695,9 @@ A StreamID of zero (0x0) is reserved and used for connection-level flow control
 frames ({{flow-control}}); the StreamID of zero cannot be used to establish a
 new stream.
 
-StreamID 1 (0x1) is reserved for the crypto handshake.  StreamID 1 MUST NOT be
-used for application data, and MUST be the first client-initiated stream.
+StreamID 1 (0x1) is reserved for the cryptographic handshake.
```

Comment:
Do you have `fill-column` set to 80, or is it smaller?  If I remember rightly, emacs defaults to 76 or something like that.  `M-x customize-variable fill-column`